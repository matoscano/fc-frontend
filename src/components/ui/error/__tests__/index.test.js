import React from "react";
import { render } from "@testing-library/react";
import Error from "../index";

it("renders a Error", () => {
  const { container } = render(<Error />);
  expect(container).toMatchSnapshot();
});
