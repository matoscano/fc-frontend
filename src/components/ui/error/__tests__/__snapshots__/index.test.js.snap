// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a Error 1`] = `
<div>
  <section
    class="sc-AxjAm jhiTqR"
  >
    <div
      class="sc-AxirZ febGTa"
    >
      <div
        class="sc-AxhCb ccMvAG"
      >
        <img
          alt="Error img"
          src="error.png"
          title="Error img"
        />
      </div>
      <div
        class="sc-AxiKw kdkRpp"
      >
        Something went wrong...
      </div>
    </div>
  </section>
</div>
`;
